lib/qa/authorities/linked_data/search_query.rb

```diff
@@ -179,18 +184,21 @@ def wrap_labels(labels)
         end
 
         def append_data_outside_results(results)
-          return results unless performance_data?
+          return results unless performance_data? || response_header?
           full_results = {}
           full_results[:results] = results
-          full_results[:performance] = performance(results)
+          full_results[:performance] = performance(results) if performance_data?
+          full_results[:response_header] = response_header(results) if response_header?
           full_results
         end
 
         def performance(results)
-          normalized_size = results.to_s.size
-          fetched_size = full_graph.triples.to_s.size
```

Comment:
per request in PR #282, these calculations were moved to performance_data_service